.github/workflows/moonbit.yml

```diff
@@ -0,0 +1,47 @@
+name: CI-moonbit
```

Comment:
If you're ok with it I'd personally prefer to avoid adding this to our CI. Contributors won't understand that by default this CI isn't necessary for landing a PR and to ignore it if it's failing.

Reply:
Sorry for the late reply. I removed the CI.